packages/foreman/nodejs-jed/nodejs-jed.spec

```diff
@@ -21,22 +28,24 @@ ExclusiveArch: %{nodejs_arches} noarch
 
 %install
 mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cp -pfr Makefile %{buildroot}%{nodejs_sitelib}/%{npm_name}
```

Comment:
Is this really needed? I also left out test in some places because it contained package.json files that confused `nodejs-packaging` auto-requires.

Reply:
*shrug* it came with the auto generation :)